### ISSUE TITLE
pretrained model 

### ISSUE THREAD

**Issue by zeduchen** [HelloRicky123/Siamese-RPN#3]
Hello, thank you for open source your code. I want to run your code. Which pretrained model does your code use? Can you give me the download URL?

**Comment by HelloRicky123** [comment]
songdejia provides a pretrained model https://github.com/songdejia/Siamese-RPN-pytorch#download-pretrained-model-on-vid-with-690000-image-pairs

**Comment by zeduchen** [comment]
Thank you! But this pre-training model is not the same as your AlexNet.

**Comment by MrYANG23** [comment]
it  seems like that the pretrained model of songdejia dont have the batchnormaliztion

**Comment by HelloRicky123** [comment]
During my training getting the 0.54 AUC, I use these code to load the former three layers of Li Bo's trained model, for they say these layers are frozen. 
"    if config.pretrained_model:
        print("pre init checkpoint %s" % config.pretrained_model)
        checkpoint = torch.load(config.pretrained_model)
        model_dict = model.state_dict()
        keys_former3conv = ['featureExtract.0.weight', 'featureExtract.0.bias', 'featureExtract.4.weight',
                            'featureExtract.4.bias', 'featureExtract.8.weight', 'featureExtract.8.bias', ]
        checkpoint = {k: v for k, v in checkpoint.items() if k in keys_former3conv}
        model_dict.update(checkpoint)
        model.load_state_dict(model_dict)
        for k, v in model.named_parameters():
            if k in keys_former3conv:
                v.requires_grad = False
        del checkpoint
        torch.cuda.empty_cache()
        print("pre inited checkpoint")"

**Comment by HelloRicky123** [comment]
And thanks to Li Bo's help, they give out the pretrained model on Imagenet, this is the address: https://drive.google.com/drive/folders/1HJOvl_irX3KFbtfj88_FVLtukMI1GTCR

**Comment by zeduchen** [comment]
Thank you!

**Comment by MrYANG23** [comment]
thank you !
